spaceship/docs/iTunesConnect.md

```diff
@@ -231,29 +227,13 @@ build.train_version           # => "0.9.21" (the version number)
 build.install_count           # => 1
 build.crash_count             # => 0
 
-build.testing_status          # => "Internal" or "External" or "Expired" or "Inactive"
+build.internal_state         # => testflight.build.state.testing.ready
+build.external_state         # => testflight.build.state.submit.ready
 ```
 
-You can even submit a build for external beta review
+You can even submit a build for external beta review (after you have set all necessary metadata - see above)
 ```ruby
-parameters = {
-  changelog: "Awesome new features",
-  description: "Why would I want to provide that?",
-  feedback_email: "contact@company.com",
-  marketing_url: "http://marketing.com",
-  first_name: "Felix",
-  last_name: "Krause",
-  review_email: "contact@company.com",
-  phone_number: "0123456789",
-  significant_change: false,
-
-  # Optional Metadata:
-  privacy_policy_url: nil,
-  review_user_name: nil,
-  review_password: nil,
-  encryption: false
-}
-build.submit_for_beta_review!(parameters)
+build.submit_for_testflight_review!
```

Comment:
I wonder if there should be a separate document explaining testflight module - since this method is from Spaceship::TestFlight::Build, as opposed to Spaceship::Tunes::Build - which was there previously.